docs/iris/src/developers_guide/release.rst

```diff
@@ -0,0 +1,66 @@
+Release
```

Comment:
To improve the look of the developer-guide table of contents, this would be better titled "Releases".